docs/advanced/exceptions.rst

```diff
@@ -79,6 +79,20 @@ This call creates a Python exception class with the name ``PyExp`` in the given
 module and automatically converts any encountered exceptions of type ``CppExp``
 into Python exceptions of type ``PyExp``.
 
+It is possible to specify base class for the exception using the third 
+parameter, a pointer to `PyObject`:
+
+.. code-block:: cpp
+
+    py::register_exception<CppExp>(module, "PyExp", PyExc_RuntimeError);
+
+Then `PyExp` can be caught both as `PyExp` and `RuntimeError`.
```

Comment:
Does this need to be explicitly specified? This is really just the basic way Python handles an exception hierarchy. (Fine to keep it though, if you want.)

Reply:
I'd like to keep it, even though it's basic (makes sure there is common understanding)

Reply:
OK, fine with me.